portafly/src/auth/AuthContext.tsx

```diff
@@ -1,4 +1,5 @@
 import React, { createContext, useContext, useState } from 'react'
+import { getToken, setToken } from 'utils'
 
 type AuthToken = string | null
```

Comment:
If the AuthToken is going to be the result of `JSON.parse` we should change this to object (eventually JWT):
```suggestion
type AuthToken = Object | null
```